.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: Build docker image
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: brpaz/hadolint-action@v1.2.1
+        name: Lint
+        with:
+          dockerfile: Dockerfile
+      -
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GH_PACKAGE_TOKEN }}
+      -
+        name: Build and push
```

Comment:
can we the action used in the `app` repo https://github.com/simple-login/app/blob/c61dd9dec6460eaa84c55b44e437d1c6991fd3a0/.github/workflows/main.yml#L52 ?